src/codegen/compile.ml

```diff
@@ -789,6 +812,9 @@ module RTS = struct
     E.add_func_import env "rts" "memcmp" [I32Type; I32Type; I32Type] [I32Type];
     E.add_func_import env "rts" "version" [] [I32Type];
     E.add_func_import env "rts" "parse_idl_header" [I32Type; I32Type; I32Type; I32Type; I32Type] [];
+    E.add_func_import env "rts" "idl_sub_buf_words" [I32Type; I32Type] [I32Type];
+    E.add_func_import env "rts" "idl_sub"
+      [I32Type; I32Type; I32Type; I32Type; I32Type; I32Type; I32Type; I32Type; I32Type] [I32Type];
```

Comment:
```suggestion
      (i32s 9) [I32Type];
```

Reply:
Oh, sorry. How about
```suggestion
      (Lib.List.make 9 I32Type);
```

Reply:
I guess there is too much code in the world already...